Docs: massive rework of Access control chapter (#5566)
WeblateOrg/weblate: docs/workflows.rst

Translation access
------------------

> The :ref:`access control <access-control>` in general is not much discussed in

Reviewer: ```suggestion
The :ref:`access control <access-control>` is not discussed in detail as a whole in
```

> the workflows as most of its option can be applied to any workflow. Please

Reviewer: ```suggestion
the workflows, as most of its options can be applied to any workflow. Please
```

> consult that documentation for information on how to manage access to

Reviewer: ```suggestion
consult the respective documentation on how to manage access to
```
